data/state/peerAccount.go

```diff
@@ -354,14 +368,60 @@ func (pa *PeerAccount) AddToAccumulatedFees(value *big.Int) error {
 
 // ResetAtNewEpoch will reset a set of values after changing epoch
 func (pa *PeerAccount) ResetAtNewEpoch() error {
-	entry, err := NewPeerJournalEntryAccumulatedFees(pa, pa.AccumulatedFees)
+	entryAccFee, err := NewPeerJournalEntryAccumulatedFees(pa, pa.AccumulatedFees)
 	if err != nil {
 		return err
 	}
 
-	pa.accountTracker.Journalize(entry)
+	pa.accountTracker.Journalize(entryAccFee)
 	pa.AccumulatedFees = big.NewInt(0)
 
+	err = pa.accountTracker.SaveAccount(pa)
```

Comment:
with current design there is need to save it for every pass. as at rollback it will go step by step through all journalized entries and will try to restore the state according to hash

Reply:
ok